src/security/PeerConnector.cc

```diff
@@ -763,16 +763,23 @@ Security::PeerConnector::resumeNegotiation()
     auto lastError = suspendedError_; // may be reset below
     suspendedError_ = nullptr;
 
+    negotiateAfterRevalidateCertificates(*lastError);
```

Comment:
This pointer may be nil, right?

Reply:
Sorry, I was going to delete this wrong change request and forgot. The pointer cannot be nil. Resolving.